src/content/docs/release-notes/docs-release-notes/docs-7-26-2024.mdx

```diff
@@ -0,0 +1,47 @@
+* With a recent relaunch of Pathpoint, we've made significant updates to related docs, but especially [introduction to New Relic Pathpoint](/docs/new-relic-solutions/business-observability/intro-pathpoint) and [install Pathpoint](/docs/new-relic-solutions/business-observability/install-pathpoint).
+* Updated [PHP agent installation: Ubuntu and Debian](/docs/apm/agents/php-agent/installation/php-agent-installation-ubuntu-debian) to remove `apt-key` due to the way it compromises trust in package signing.
+
+### Minor changes
+
+* Update to [login and password issues](/docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems/#social-login-error) give you some info on what to do when your social login fails.
+* Updated the screenshot at the top of [get started with infrastructure monitoring](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/) to match the current UI.
+* Removed invalid `webelement` methods from [synthetic scripted browser reference (Chrome 100 and higher)](/docs/synthetics/synthetic-monitoring/scripting-monitors/synthetic-scripted-browser-reference-monitor-versions-chrome-100). (Updated in response to what's shown in the [Selenium `WebElement](https://www.selenium.dev/selenium/docs/api/javascript/WebElement.html) class definition.)
+* Clarified in [compatibility and requirements for the Node.js agent](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent) that New Relic typically ends supports for a Node.js version 2-4 months after [the published EOL date](https://nodejs.org/en/about/previous-releases).
+* In [infrastructure inventory UI page](/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure), added note that APM metadata is created in the ingest pipeline when infrastructure and APM agents are running on the same host.
+* In [import, export, add, and share dashboards and charts](/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/#export-pdf-limitations) added a section describing some of the limitations when exporting a dashboard to PDF.
+* Clarified reduce data ingest and metrics in [OpenTelemetry for Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel). Also, added a new diagram.
+* Microsoft dropped support Windows Server 2012 and 2012 R2 on October 10, 2023. We've updated [requirements for the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/requirements-infrastructure-agent) to reflect that.
+* Updated the video on [intro to logs in context](/docs/logs/logs-context/logs-in-context/).
+* Updated "do not translate" tags across our site to improve the translation accuracy of UI and non-UI text strings.
```

Comment:
```suggestion
  * Updated `<DoNotTranslate>` tags across our site to improve the translation accuracy of UI and non-UI text strings.
```

Reply:
Maybe actually include the tag name instead of putting it in quotes

Reply:
@ally-sassman Isn't the tag `<DNT>` now?

Reply:
Ah I see, so maybe specify what it was and what it is now (i.e., <DoNotTranslate> is now <DNT>)

This could be useful for contributors to know what DNT is now

Reply:
But not a hard suggestion, feel free to discard this